inst/include/cpp11/doubles.hpp

```diff
@@ -84,7 +84,6 @@ template <>
 inline r_vector<double>::r_vector(std::initializer_list<named_arg> il)
     : cpp11::r_vector<double>(safe[Rf_allocVector](REALSXP, il.size())),
       capacity_(il.size()) {
-  protect_ = detail::store::insert(data_);
```

Comment:
In all the specialization files, this was updating the writable `protect_`, which no longer exists